src/main/java/com/conveyal/taui/AnalysisServer.java

```diff
@@ -81,16 +81,17 @@ public static void main (String... args) {
             // Default is JSON, will be overridden by the few controllers that do not return JSON
             res.type("application/json");
 
-            // Log each API request
-            LOG.info("{} {}", req.requestMethod(), req.pathInfo());
-
-            if (!AnalysisServerConfig.offline) {
+            if (AnalysisServerConfig.auth0ClientId != null && AnalysisServerConfig.auth0Secret != null) {
                 handleAuthentication(req, res);
             } else {
+                LOG.warn("No Auth0 credentials given, set accessGroup and email automatically");
```

Comment:
The warning message sounds imperative and might confuse someone in the future (like "You should set the accessGroup automatically.") A better message might be "No Auth0 credentials were supplied, setting accessGroup and email to placeholder defaults".